presto-spi/src/main/java/com/facebook/presto/spi/security/AccessControlContext.java

```diff
@@ -32,8 +32,19 @@ public class AccessControlContext
     private final WarningCollector warningCollector;
     private final RuntimeStats runtimeStats;
     private final Optional<QueryType> queryType;
+    private final Optional<String> catalogName;
```

Comment:
sorry, one more nit: either catalogName and schemaName or catalog and schema (and corresponding changes to constructor and getter).  My preference is catalog and schema